src/patternfly/components/InlineEdit/inline-edit.scss

```diff
@@ -0,0 +1,193 @@
+.pf-c-inline-edit {
+  // Group
+  --pf-c-inline-edit__group--item--MarginRight: var(--pf-global--spacer--sm);
+
+  // Value group
+  --pf-c-inline-edit__group--m-value-group--item--MarginRight: var(--pf-global--spacer--sm);
+
+  // Input group
+  --pf-c-inline-edit__group--m-input-group--item--MarginRight: var(--pf-global--spacer--sm);
+
+  // Action group
+  --pf-c-inline-edit__action-group--MarginTop: var(--pf-global--spacer--xl);
+  --pf-c-inline-edit__group--m-action-group--item--MarginRight: var(--pf-global--spacer--md);
+  --pf-c-inline-edit__group--m-input-group--m-action-group--item--MarginRight: 0;
+
+  // Icon group
+  --pf-c-inline-edit__group--m-icon-group--item--MarginRight: 0;
+
+  // Toggle
+  --pf-c-inline-edit--m-active__toggle--Color: var(--pf-global--Color--100);
+
+  // Save button
+  --pf-c-inline-edit__action--m-save--m-valid--c-button--m-plain--Color: var(--pf-global--link--Color);
+
+  // Hidden
+  .pf-m-hidden {
+    display: none;
+    visibility: hidden;
+  }
+
+  .pf-m-display-inline-flex {
+    display: inline-flex;
+  }
+
+  // Align right
+  .pf-m-align-right {
+    margin-left: auto;
+  }
+
+  /* stylelint-disable selector-max-compound-selectors */
+  [class*="pf-m-grid"] [data-label] {
+    .pf-c-check,
+    .pf-c-radio {
+      input {
+        grid-column: 1;
+      }
+    }
+  }
+  /* stylelint-enable */
+}
+
+.pf-c-inline-edit__group {
+  display: flex;
+
+  > * {
+    margin-right: var(--pf-c-inline-edit__group--item--MarginRight);
+  }
+
+  // Value group
+  &.pf-m-value-group {
+    --pf-c-inline-edit__group--item--MarginRight: var(--pf-c-inline-edit__group--m-value-group--item--MarginRight);
```

Comment:
Do you need to re-assign this variable here if they both have the same value of `sm` ? I think you could get rid of the var altogether

Reply:
It's not necessary, but allows for the value to be customized/themed.